src/data/repositories/TrainingModuleDefaultRepository.ts

```diff
@@ -57,7 +87,8 @@ export class TrainingModuleDefaultRepository implements TrainingModuleRepository
             );
 
             const currentUser = await this.config.getUser();
-
+            //I think this show all the modules on the main screen 
+            //We would filter by the modules that have TrainingModule.installed === true 
```

Comment:
No, it's fine to return all modules here. We should filter them on the ``HomePage`` component